src/erlcloud_s3.erl

```diff
@@ -714,11 +719,13 @@ sign_request(Expire_time, BucketName, Key, Method, Mime, Config)
         undefined -> "";
         SecurityToken -> "x-amz-security-token:" ++ SecurityToken ++ "\n"
     end,
-    To_sign = lists:flatten([Method, "\n\n", Mime, "\n", Expires, "\n", SecurityTokenToSign, "/", BucketName, "/", Key]),
+    MetaDataToSign = lists:sort(
+                       ["x-amz-meta-" ++ string:to_lower(MKey) ++ ":" ++ MValue ++ "\n"
```

Comment:
According to [this](http://docs.aws.amazon.com/general/latest/gr/sigv4-signed-request-examples.html), the values have to be lowercased, not only the key.
(And I suggest to sort all `x-amz-` together so we don't rely on the order of `MetaDataToSign` and `SecurityTokenToSign` below)